packages/parser-html/README.md

```diff
@@ -0,0 +1,60 @@
+# HTML parser (`@hint/parser-html`)
+
+The `HTML` parser is built on top of `[jsdom][jsdom]` so hints can
+analyze `HTML` files.
+
+Note: This parser is not needed if using a [connector][connectors]
+which connects to a browser as the browser's DOM will be used to
+generate events instead.
```

Comment:
The `jsdom` connector doesn't need it neither. Maybe reformulate to say it's only required by `local`?